_data/tables/transform-high-performing-org.yml

```diff
@@ -39,22 +44,60 @@
       data:
         list: false
         fr: 
-        en: Establish communication plan and change management for continuous improvement in (and around) IITB
+        en: Develop a self assessement tool based on capabilities of high performing organizations and PwC recommendations
     - col:
       data:
         list: false
         fr: 
-        en: 
+        en: To help teams understand the current condition and set measurable targets, the self assessment tool will guide teams toward ways to improve based on an evaluation of diferent capabilities linked to high performing organizations.
     - col:
       data:
         list: true
         items:
         - item:
-            fr: <strong>BOP</strong>
-            en: <strong>BOP</strong>
+            fr: <strong>Stratégie TI</strong>
+            en: <strong>IT Strategy</strong>
+        - item:
+            fr: R&P
+            en: R&P
+    - col:
+      data:
+        list: false
+        fr: 2020-2021 Q2
+        en: 2020-2021 Q2
+  - row:
+    headers: false
+    cols:
+    - col:
+      data:
+        list: false
+        fr: Gouvernance
+        en: Governance
+    - col:
+      data:
+        list: false
+        fr: 
+        en: Define how DGs will guide their teams to achieve the goals of the Guiding Policy
```

Comment:
Define how DGs will guide their teams to achieve the goals of the Guiding Policy
==>
DGs shall define their own continuous improvement metrics (tied to that of the CIO)

Reply:
The guiding policy says: "IITB teams seek to understand the current condition, set measurable targets for a future date and experiment with ways to achieve these targets until the future date is reached."  That includes DGs.

This was an actions for CIOEC to figure out how to move forward with the strategy

Reply:
Isnt that what we're supposed to be doing? Figuring out how to move forward with the strategy?